packages/popmotion-pose/docs/api/react/posed.md

```diff
@@ -98,10 +135,10 @@ const sidebarProps = {
   closed: { x: '-100%' }
 }
 
-const Sidebar = styled(posed.nav(sidebarProps))`
+const Sidebar = posed(styled.nav`
```

Comment:
Leave this as before - I'd like to encourage the previous syntax because it leaves the complexity on Styled Components.